Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -261,8 +261,17 @@ namespace MultiplayerSample
 #if AZ_TRAIT_SERVER
     void NetworkMatchComponentController::EndRound()
     {
-        ModifyRoundNumber()++;
-        SetRoundTime(RoundTimeSec{ GetRoundDuration() });
+        uint16_t roundNumber = GetRoundNumber() + 1;
+
+        // We need to do this whether or not we're going beyond the number of total rounds so that
```

Comment:
Good call out, ran into that confusion too. 
Will need to merge with... https://github.com/aws-lumberyard-dev/o3de-multiplayersample/pull/144
We'll race to merge 1st :O

Reply:
I win!!!